src/reflow/mod.rs

```diff
@@ -59,27 +57,20 @@ impl Checker for Reflow {
                         });
                     // remove last newline
                     let _ = new_lines.pop();
-                    // @todo that's too easy
-                    if new_lines.len() == chunk.as_str().lines().count() {
+                    // no reflow requried
+                    // @todo still looks quite fishy, rethink those replace() calls
+                    if new_lines.join("") == chunk.as_str().replace("\n", "").replace("   ", " ") {
```

Comment:
Joining without newlines is kinda aquard - it doesn't really allow for detection of the new newline placements - you remove all - so the discriminant power of this line is pretty much zero iiuc

Reply:
I basically compare the strings without any newlines.. they should be equal every single time!

I'm thinking of comparing the line numbers and the line lengths, that should be enough right?